apps/web/src/api/hooks/useMakePrimaryIntegration.ts

```diff
@@ -0,0 +1,54 @@
+import { useMutation, useQueryClient, UseMutationOptions } from '@tanstack/react-query';
+
+import { errorMessage } from '../../utils/notifications';
+import type { IntegrationEntity } from '../../pages/integrations/types';
+import { successMessage } from '../../utils/notifications';
+import { QueryKeys } from '../query.keys';
+
+export const useMakePrimaryIntegration = (
```

Comment:
this is the hook that is used when the user clicks the "Make primary" button in the modal... it's not integrated yet with the BE endpoint